opentracing-spring-web/src/test/java/io/opentracing/contrib/spring/web/interceptor/TracingHandlerInterceptorTest.java

```diff
@@ -0,0 +1,47 @@
+package io.opentracing.contrib.spring.web.interceptor;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import io.opentracing.contrib.web.servlet.filter.TracingFilter;
+
+public class TracingHandlerInterceptorTest {
+
+    @Test
+    public void testHasSpanStarted() {
+        HttpServletRequest request = Mockito.mock(HttpServletRequest.class);
+        Mockito.when(request.getAttribute(TracingFilter.SERVER_SPAN_CONTEXT)).thenReturn("");
```

Comment:
`.thenReturn("")` works here I would prefer to pass an `Object` there. Maybe better it would be to change 

`hasSpanStarted` to `httpServletRequest.getAttribute(TracingFilter.SERVER_SPAN_CONTEXT) instanceOf SpanContext`